components/frontend/src/metric/status.js

```diff
@@ -23,6 +25,22 @@ export const STATUS_COLORS_RGB = {
     informative: "rgb(0,165,255)",
     unknown: "rgb(245,245,245)",
 }
+export const STATUS_COLORS_MUI = {
+    target_not_met: red[700],
+    target_met: green[600],
+    near_target_met: orange[300],
+    debt_target_met: grey[500],
+    informative: blue[500],
+    unknown: grey[300],
+}
+export const STATUS_ICONS = {
+    target_met: <Check />,
+    near_target_met: <Warning />,
+    debt_target_met: <Money />,
+    target_not_met: <Bolt />,
+    informative: <b>i</b>,
```

Comment:
There's no (i) icon?

Reply:
Surprisingly enough not. This works pretty well though.